dictionary/Dictionary.js

```diff
@@ -45,6 +45,24 @@ export default class Dictionary {
 	 *         evaluated using the provided parameters.
 	 */
 	get(key, parameters = {}) {}
+
+	/**
+	 * Tests whether the specified localization phrase exists in dictionary
+	 *
+	 * @param {string} key The key identifying the localization phrase.
+	 * @return {boolean} {@code true} if the the key (phrase) exists.
```

Comment:
It might be reasonable to use the wording "if and only if" (equivalence vs implication).